data/teammembers.js

```diff
@@ -90,20 +90,25 @@ export default {
       is_active: true,
     },
     {
-        name: 'Ashley Cho',
-        position: 'Website Manager',
-        introduction: `I enjoy being a part of Peer1on1 because I feel like I am able to help out so many people and make a real difference. It also pushes me to strive to improve myself, because it helps me realize that in order to help others reach a goal, I must proficient in that area as well.
-        My future goals are to study computer sciences.`,
-        avatar_img: Ashley,
-        is_active: true,
+      name: 'Ashley Cho',
+      position: 'Website Manager',
+      introduction: `I enjoy being a part of Peer1on1 because I feel like I am able to help out so many people and make a real difference. It also pushes me to strive to improve myself, because it helps me realize that in order to help others reach a goal, I must proficient in that area as well.
+      My future goals are to study computer sciences.`,
+      avatar_img: Ashley,
+      is_active: true,
     }, 
     {
+      name: 'Derek M',
+      position: 'Music Director',
+      avatar_img: missing_avatar,
+      is_active: true,
+    },
       name: 'Jason Zhao',
```

Comment:
The '{' is missing before this line, it should be:
{
   name: 'Jason Zhao',
...

Reply:
I fixed it and added it to develop. How do I add it to the pull request?

Reply:
Your latest change is included in this pull request, there is another error, I will check it further.